interp/Cargo.toml

```diff
@@ -51,3 +52,7 @@ proptest = "1.0.0"
 
 [dependencies.serde_with]
 version = "1.6.4"
+
+
+
+
```

Comment:
It would be nice to remove these extra blank lines (here in in the root `Cargo.toml`). I know it is not actually important, but it is helpful to keep these "changed files" pages on the GitHub pull request interface short and easier to process!